<?xml version="1.0"?>
<launch>
    <arg name="robot1_name" default="jetauto_1"/>
    <arg name="robot2_name" default="robot_2"/>

    <node pkg="tf" type="static_transform_publisher" name="map_to_robot1" args="0 0 0 0 0 0 /map $(arg robot1_name)/map 50" />
    <node pkg="tf" type="static_transform_publisher" name="map_to_robot2" args="0 0 0 0 0 0 /map $(arg robot2_name)/map 50" />

    <group ns="$(arg robot1_name)/map_merge">
        <param name="init_pose_x"   value="0.0"/>
        <param name="init_pose_y"   value="0.0"/>
        <param name="init_pose_z"   value="0.0"/>
        <param name="init_pose_yaw" value="0.0"/>
    </group>
    <group ns="$(arg robot2_name)/map_merge">
        <param name="init_pose_x"   value="0.0"/>
        <param name="init_pose_y"   value="1.0"/>
        <param name="init_pose_z"   value="0.0"/>
        <param name="init_pose_yaw" value="0.0"/>
    </group>
</launch>
